src/dev/i18n/extract_default_translations.test.js

```diff
@@ -17,29 +17,72 @@
  * under the License.
  */
 
-import { resolve } from 'path';
+import path from 'path';
 import fs from 'fs';
 import { promisify } from 'util';
+import JSON5 from 'json5';
+import { i18n } from '@kbn/i18n';
 
 import { extractDefaultTranslations } from './extract_default_translations';
 
 const readFileAsync = promisify(fs.readFile);
 const removeDirAsync = promisify(fs.rmdir);
 const unlinkAsync = promisify(fs.unlink);
 
-const PLUGIN_PATH = resolve(__dirname, '__fixtures__', 'test_plugin');
+const fixturesPath = path.resolve(__dirname, '__fixtures__', 'extract_default_translations');
+const pluginsPaths = [
+  path.join(fixturesPath, 'test_plugin_1'),
+  path.join(fixturesPath, 'test_plugin_2'),
+  path.join(fixturesPath, 'test_plugin_3'),
+];
 
 describe('dev/i18n/extract_default_translations', () => {
+  it('extracts messages to en.json', async () => {
+    await extractDefaultTranslations(pluginsPaths[0]);
```

Comment:
__nit:__ maybe just `const [pluginPath] = pluginsPaths` here and `const [, pluginPath] = pluginsPaths;` in test(s) below to make the rest of the test more readable?